earthfile2llb/interpreter.go

```diff
@@ -32,6 +32,9 @@ import (
 
 var errCannotAsync = errors.New("cannot run async operation")
 
+var filesWithCommandRenameWarning = make(map[string]bool)
+var maxCommandRenameWarnings = 3
```

Comment:
We generally avoid global variables, unless we really have no choice. It might be better to store the map in `ConvertOpts` from earthfile2llb.go? The maxCommandRenameWarnings can stay here, but it should be a `const`.